### ISSUE TITLE
Absolutely no links work in Firefox with reasonable security settings

### ISSUE THREAD

**Issue by sdunbar-wrs** [quarkusio/quarkusio.github.io#1100]
**Describe the bug**
There are no links on quarkus.io that can be clicked.

**Which pages**
Pick a page, click on a link.

Because:
18:26:05.854 Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://dpm.demdex.net/id?d_visid_ver=5.2.0&d_fieldgroup=AAM&d_rtbd=json&d_ver=2&d_orgid=945D02BE532957400A490D4C%40AdobeOrg&d_nsid=0&d_mid=92180643808121403253983962995112937483&d_coppa=true&ts=1629159965840. (Reason: CORS request did not succeed).

Try an anchor tag with a local href.  It's way simpler.

**Comment by maxandersen** [comment]
I use firefox and have zero such problems.

Can you elaborate what "reasonable security settings" mean for you ?

**Comment by maxandersen** [comment]
btw. i do see the cross-origin error but im not seeing linked blocked etc. so would love more specific details.

possible related past issues #236 and #209

**Comment by maxandersen** [comment]
@insectengine are you aware of us missing or changing some CORS setup?

**Comment by sdunbar-wrs** [comment]
Of course.  I'm using the "Strict" settings:
![image](https://user-images.githubusercontent.com/74786781/129800342-8efcd253-e8db-4ffc-832d-76a169665419.png)

Note that with "Standard" settings the site works as expected.  I'm on Firefox 91.0 on Ubuntu.

**Comment by maxandersen** [comment]
Well I believe we should work with strict. At least we did before so let's see why we are causing cors when we should mnt.

**Comment by insectengine** [comment]
https://stackoverflow.com/questions/62478071/firefox-enhanced-privacy-protection-is-blocking-datalayer-push-to-google-tag-man

I don't know a way around this issue.

**Comment by maxandersen** [comment]
Turn of the conversion tracking for links should work or ?

**Comment by holly-cummins** [comment]
Is this still an issue? I think we no longer use GTM.

**Comment by insectengine** [comment]
> Is this still an issue? I think we no longer use GTM.

Correct, no longer in place. The only analytics tracking is matomo. I say close this.

**Comment by holly-cummins** [comment]
I tried in Firefox with Strict and all seems well.
